v2/spiffetls/tlsconfig/trace.go

```diff
@@ -0,0 +1,25 @@
+package tlsconfig
+
+import (
+	"crypto/tls"
+	"time"
+)
+
+type GetTLSCertificateStart struct {
+	Start time.Time
+	End   time.Time
+	Cert  *tls.Certificate
+	Err   error
```

Comment:
These fields aren't used.

Reply:
Oups! Forgot to put the code that handles this stuff!